macros/utils/cross_db_utils/current_timestamp.sql

```diff
@@ -1,16 +1,48 @@
 {% macro current_timestamp() -%}
+    {{ adapter.dispatch('current_timestamp','elementary')() }}
+{%- endmacro %}
+
+{% macro default__current_timestamp() -%}
+    {% set macro = dbt.current_timestamp_backcompat or dbt_utils.current_timestamp %}
+    {% if not macro %}
+        {{ exceptions.raise_compiler_error("Did not find a `current_timestamp` macro.") }}
+    {% endif %}
+    {{ return(macro()) }}
+{%- endmacro %}
+
+{% macro databricks__current_timestamp() -%}
```

Comment:
TBH I don't understand why this works for Databricks. 
The adapter does not include a macro named `current_timestamp_backcompat`